crates/subspace-networking/src/protocols/connected_peers.rs

```diff
@@ -51,6 +51,9 @@ use tracing::{debug, trace};
 /// Represents different states of a peer permanent connection.
 #[derive(Debug, Clone, PartialEq, Eq)]
 enum ConnectionState {
+    /// Indicates that a connection is expected to be established to this peer.
+    ToConnect { peer_address: Multiaddr },
```

Comment:
`PendingConnection` seems like an "established name" for a networking protocol state.

Reply:
Isn't `Connecting` also `Pending` essentially?